### ISSUE TITLE
test is stuck at (jepsen/sycnhronize ) due to not configuring all the nodes.

### ISSUE THREAD

**Issue by amateurInventer** [aphyr/jepsen#99]
Hi,
I am using jepsen framework inside docker container. Basically there is one docker in which I setup the jepsen testing framework. Further according to nodes requirement for the test I create dockers containers inside it. All my tests gets stuck intermittently in following way:

INFO  jepsen.os.debian - :n2 setting up debian
INFO  jepsen.os.debian - :n1 setting up debian
INFO  jepsen.os.debian - :n3 setting up debian
INFO  jepsen.os.debian - Installing #{rsyslog}
INFO  jepsen.os.debian - Installing #{rsyslog}
INFO  jepsen.os.debian - Installing #{rsyslog}
INFO  tacc.core - :n2 not tearing down system
INFO  tacc.core - :n3 not tearing down system
INFO  tacc.core - :n1 not tearing down system
INFO  tacc.core - :n1 tore down
INFO  tacc.core - :n2 tore down
INFO  tacc.core - :n3 tore down
INFO  jepsen.os.debian - setting up :optumsoft apt repo
INFO  jepsen.os.debian - setting up :optumsoft apt repo
INFO  jepsen.os.debian - setting up :optumsoft apt repo
INFO  jepsen.os.debian - Installing #{\<some packages\>}
INFO  jepsen.os.debian - Installing #{\<some packages\>}
INFO  jepsen.os.debian - Installing #{\<some pacakges\>}
INFO  tacc.core - :n1 done install
INFO  tacc.core - :n1 {scheduler.json [:n1]}
INFO  tacc.core - inside :n1 scheduler.json
INFO  tacc.core - :n1 done configure
INFO  tacc.core - :n1 system set up

Basically package installation step does not start on one of the nodes and test gets stuck at (jepsen/synchronize )
Is there any thing I should take care of in my test?

Thanks

**Comment by aphyr** [comment]
You'll have to figure out what's stuck and get it unstuck. Not sure what else to tell you. :/
